content/client-lib-development-guide/features.textile

```diff
@@ -326,14 +333,14 @@ h3(#realtime-connection). Connection
 ** @(RTN13c)@ Will fail if a @HEARTBEAT@ @ProtocolMessage@ is not received within the "default realtime request timeout":#defaults
 * @(RTN14)@ @Connection@ opening failures:
 ** @(RTN14a)@ If an API key is invalid, then the connection will enter the @FAILED@ state and the @Connection#errorReason@ will be set on the @Connection@ object as well as the emitted @ConnectionStateChange@
-** @(RTN14b)@ If a connection request fails due to an @ERROR@ @ProtocolMessage@ being sent to the client containing a token error (@statusCode@ value of 401 and error @code@ value in the range @40140 <= @code < 40150@), then if the token is renewable, the error should not be emitted, a single attempt to create a new token will be made and a new connection attempt will be initiated using the newly created token. If the attempt to create a new token fails, or the subsequent connection attempt fails due to another token error, then the connection will transition to the @FAILED@ state and the @Connection#errorReason@ will be set
+** @(RTN14b)@ If a connection request fails due to an @ERROR@ @ProtocolMessage@ being sent to the client containing a token error (@statusCode@ value of 401 and error @code@ value in the range @40140 <= code < 40150@), then if the token is renewable, the error should not be emitted, a single attempt to create a new token will be made and a new connection attempt will be initiated using the newly created token. If the attempt to create a new token fails, or the subsequent connection attempt fails due to another token error, then the connection will transition to the @FAILED@ state and the @Connection#errorReason@ will be set
```

Comment:
@paddybyers see https://github.com/ably/docs/pull/133/commits/175e6ccdfe3f68e595a5eadddb8146264e91764c, it's just a fix to a typo in those error code ranges